### PR TITLE
Docs Update

### PR DESCRIPTION
**Pull request by markymark-26**
# Description

Merges content changes to docs into main on behalf of BLIP-11 effort following the changes from #466. 

## Related Issue

[BLIP-11](https://github.com/eea-oasis/baseline-blips/issues/21)

## Motivation and Context

This change is required to provide further clarity to the docs, which includes the updates to links, phrasing, grammar, and content.

## How Has This Been Tested

Only changes to docs included. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Request to be added as a Code Owner/Maintainer

## Checklist

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.
- [X] I commit to abide by the Responsibilities of Code Owners/Maintainers.

### CODE REVIEW

**Review by Therecanbeonlyone1969 — APPROVED** [review]
Not a fan of huge PRs but LGTM

**Review by skarred14 — APPROVED** [review]

File: community/contributors.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+Core Developers meet and discuss issues virtually via the #maintainers slack room in the [baseline slack](https://ethereum-baseline.slack.com).
+
+### Regular Meetings <a href="#weekly-meetings" id="weekly-meetings"></a>
+
+There are weekly _Core Developers meetings_ where members can discuss plans and issues related to the project, updates, release planning, and other related topics. Anyone may attend these meetings, but the primary participants are core developers. Core developers are required to produce meeting summaries and document decisions.&#x20;
+
+Meeting signups are maintained on the "Join the Community" page on https://baseline-protocol.org.
+
+## How to stop being a core developer? <a href="#how-to-stop-being-a-maintainer" id="how-to-stop-being-a-maintainer"></a>
```
Comment (Manik-Jain) [inline community/contributors.md:97]: This seems to be a misleading statement. I mean we don't want them to do any of these like being disrespectful to others, etc. 

`Suggestion : `
How to stop being a core developer? 
You notify the other core developers  you would like to relinquish your core developer status.

What can take away your Core Dev status?
* You stop reviewing PR's, responding to messages, answering emails, and/or generally ghost the project.
* You are disrespectful towards anyone in the community and/or involved in the project.
* You are disruptive to the general process of maintaining the project, meetings, discussions, issues, or other.

**Comment by markymark-26** [comment]
Changes have been added to "How to Stop Being a Baseline Core Developer?" section per reviewed comments.

**Review by fosgate29 — APPROVED** [review]
LGTM

**Review by Manik-Jain — APPROVED** [review]

**Pull request merged**